### ISSUE TITLE
Fish Tank

### ISSUE THREAD

**Issue by BigBitsaByte** [Michiel1973/amazonia#83]
Went out, used the animal net (red one) captured 1 fish and 1 jellyfish.
Brought them back to put in the tank.

When placing them they turned into these. 
![tank](https://user-images.githubusercontent.com/63552914/79603499-74cbb100-80ba-11ea-9315-39d525591334.png)

**Comment by Michiel1973** [comment]
yeah, that's another issue caused by us using two separate NPC frameworks... one for petz (which has the fishtank) and one for the water animals and various underground monsters.

Can't really remove either, so thinking of just killing off the net.
